test/run_tests.py

```diff
@@ -770,7 +770,7 @@ def convert_tests_for_flatbuffers(
             )
             continue
         # todo: 300 understand why is it failing
-        # test 189, 312, 316, 318, 351, 438, 394 and 319 depend on dsjson parser behaviour
+        # test 189, 312, 316, 318, 319, 351, 367, 368, 394, 438, 456, 457 depend on dsjson parser behaviour
```

Comment:
367/368 dont seem to be using dsjson?

Reply:
https://github.com/VowpalWabbit/vowpal_wabbit/blob/master/test/core.vwtest.json#L4736 I updated those two tests with the IGL reduction PR, but didn't update this comment.